.github/ISSUE_TEMPLATE/Question.md

```diff
@@ -11,5 +11,5 @@ so we recommend using our other community resources instead of asking here 👍.
 
 If you have a support request or question please submit them to one of these resources:
 
-* [Terraform community resources](https://www.terraform.io/docs/extend/community/index.html)
```

Comment:
I think having this point directly to the Terraform section of the community forum would help with discovery 

https://discuss.hashicorp.com/c/terraform-core

Reply:
Originally had it, changed it back based on something you had in my notes. Has been updated.